super-csv/src/main/java/org/supercsv/prefs/CsvPreference.java

```diff
@@ -430,21 +448,38 @@ public Builder maxLinesPerRow(final int maxLinesPerRow) {
 		}
 		
 		/**
-		 * Uses an EmptyColumnParsing to determine whether empty String (i.e. "") should be read as empty string instead as null. 
+		 * Uses an EmptyColumnParsing to determine whether empty String (i.e. "") should be read as empty string instead as null.
 		 * The default is <tt>ParseEmptyColumnsAsNull</tt>.
-		 * 
+		 *
 		 * @since 2.4.1
 		 * @param emptyColumnParsing
 		 *            the emptyColumnParsing
 		 * @return the updated Builder
-		 */		
+		 */
 		public Builder setEmptyColumnParsing(final EmptyColumnParsing emptyColumnParsing) {
 			if( emptyColumnParsing == null ) {
 				throw new NullPointerException("emptyColumnParsing should not be null");
 			}
 			this.emptyColumnParsing = emptyColumnParsing;
 			return this;
 		}
+
+		/**
+		 * Value indicating the character to use for escaping a quote char.
+		 *
+		 * NOTE: this field is only supported for reading data.  In writing data this value is
+		 * ignored and data is written in RFC-compliant form, meaning the quote character is also
+		 * the quote escape character.
+		 *
+		 * @since X
```

Comment:
Yes, definitely it should be defined what X, Y and Z mean in our versioning schema - X.Y.Z

I think if we would reimplement main classes and changed API in such way that it is hard to recognise them then we could increase part  "X". "Y" - tells that some bugs were fixed and/or new features had been introduced. "Z" - part is just in case if we forgot to add some documentation or fixed some typos. 

For me, looking at the current changes, we reached "Y" level. But of course maybe something new appear in few days and we will be forced to change it to v3. 😉